dmcontent/html.py

```diff
@@ -81,12 +81,61 @@ def to_summary_list_rows(questions, *, filter_empty=True, **kwargs) -> List[dict
     :param bool filter_empty: Whether or not to include unanswered questions in the rows
     """
     return [
-        {"key": {"text": question.label}, "value": {"html": to_html(question, **kwargs)}}
+        to_summary_list_row(question, **kwargs)
         for question in questions
         if not (question.is_empty and filter_empty)
     ]
 
 
+def to_summary_list_row(question, *, action_link=None, **kwargs) -> List[dict]:
+    """Convert a QuestionSummary into a row for govukSummaryList.
+
+    This method expects a QuestionSummary.
+
+    `kwargs` are passed to `to_html`.
+
+    :param string action_link: A link for the row's action
+    """
+    if action_link is not None:
+        try:
+            question.empty_message
+        except AttributeError:
+            question.empty_message = "Not answered"
```

Comment:
We can replace this with `.get()`

```suggestion
        empty_message = question.get("empty_message", "Not answered")
```